doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,290 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
-need a longer lifetime, we have to copy or move the data from the ``sample``.
+To enable zero-copy inter-process communication, iceoryx uses the shared memory approach, i.e. publishers and 
+subscribers can communicate via shared memory resulting in zero-copy communication.
 
-This only allows us to get one sample at a time. Should we want to get all currently available samples we can do so
-by using an additional loop.
+Shared memory is physical memory that is made accessible to multiple processes via a mapping to a memory area in their 
+virtual address spaces.
 
+For further information have a look at our 
+[conceptual guide](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/conceptual-guide.md).
+
+### Runtime
+
+Each application that wants to use iceoryx has to instantiate its runtime, which essentially enables communication
+with RouDi.
+
+To do so, the following lines of code are required
 ```cpp
-while (keepRunning)
-{
-    while (subscriber.hasData())
-    {
-        subscriber->take()
-            .and_then([](iox::popo::Sample<const CounterTopic>& sample) {
-                CounterTopic* ptr = sample.get();
-                /* process the received data using the ptr */
-
-                /* alternatively use operator-> */
-                uint32_t counter = sample->counter;
-            })
-            .or_else([](iox::popo::ChunkReceiveResult) { /* handle the error */ });
-    }
-    // wait for new samples (either sleep or use the waitset)
-}
+iox::runtime::PoshRuntime::initRuntime("some_unique_application_name");
 ```
 
-Here we do not check whether we actually have data since we already know there is data available by calling `hasData`.
+The runtime is an object inside the user application that maps the shared memory into the user 
+application's address space. When an application terminates, the runtime cleans up all resources needed for the 
+communication with RouDi. If an application crashes, it also takes care about the clean-up. Due to our 
+[lock-free constructs](https://github.com/eclipse-iceoryx/iceoryx/tree/master/iceoryx_utils#concurrent), 
+a crash of one application does not affect the whole iceoryx system.
+
+!!! note
+    Only one runtime object per user application is allowed.
 
-### Untyped API
+### Creating service descriptions for topics
 
-The untyped API offeres similar capabilities and is hence usable in a similar way. The major difference is that
-neither publisher nor subscriber have any knowledge about the underlying type they send or receive. This means that
-the user is responsible to ensure the data is read correctly, i.e. there is no type safety guaranteed by the API
-itself.
+A ``ServiceDescription`` in iceoryx represents a topic under which publisher and subscribers can exchange data and is 
+uniquely identified by three string identifiers.
 
-#### Creating a publisher
+1. ``Group`` name
+2. ``Instance`` name
+3. ``Topic`` name
```

Comment:
should we change this in `ServiceDescription` ctor and also the `AnyService`, `InvalidIDString`, etc. constants?